okhttp-brotli/src/test/java/okhttp3/brotli/BrotliInterceptorTest.kt

```diff
@@ -21,6 +21,7 @@ import assertk.assertions.hasMessage
 import assertk.assertions.isEmpty
 import assertk.assertions.isEqualTo
 import java.io.IOException
+import kotlin.test.assertFailsWith
```

Comment:
I’m using assertFailsWith instead of assertFailure because assertFailure needs a 2nd assertion to specify the exception type

Reply:
Yeah I argued for an overload that took a reified exception type when I added `assertFailure` and lost (or, well, didn't press it too hard when given push-back). 

Maybe file a feature request and we can try again?

Reply:
https://github.com/willowtreeapps/assertk/issues/511